frontend/packages/console-app/src/components/quick-starts/QuickStartController.tsx

```diff
@@ -6,9 +6,15 @@ import { QuickStartContext, QuickStartContextValues } from './utils/quick-start-
 
 type QuickStartControllerProps = {
   quickStart: QuickStart;
+  footerClass: string;
+  updateContentRef: (ref: HTMLDivElement) => void;
```

Comment:
Rename and update type:
```suggestion
  contentRef: React.Ref<HTMLDivElement>;
```